packages/manager/src/features/CloudPulse/Utils/ReusableDashboardFilterUtils.ts

```diff
@@ -43,7 +45,7 @@ export const getDashboardProperties = (
     }),
     dashboardId: dashboardObj.id,
     duration: timeDuration ?? { unit: 'min', value: 30 },
-    resources: [String(resource)],
+    resources: filterValue[RESOURCE_ID] as CloudPulseResources[],
```

Comment:
@ankitaakamai , I think this is not going to work, did we test the contextual view as part of dev, is anything got affected?